platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -0,0 +1,485 @@
+####################################################################################################
+## Anchor Platform - Default Configuration
+##
+## Values not defined in the user config use the default values defined here.
+####################################################################################################
+
+version: 1
+
+##############################
+## Application Configuration
+##############################
+
+## @param: host_url
+## @type: string
+## @supported_prefixes: http:, https:
+## The anchor platform host url.
+#
+host_url: http://localhost:8080
+
+stellar_network:
+  ## @param: network
+  ## Whether to use testnet (TESTNET) or pubnet (PUBNET)
+  #
+  network: TESTNET
+
+  ## @param: network_passphrase
+  ## @type: string
+  ## The stellar network passphrase used to access Horizon server.
+  ## For pubnet: use 'Public Global Stellar Network ; September 2015'
+  ## For testnet: use 'Test SDF Network ; September 2015'
+  #
+  network_passphrase: Test SDF Network ; September 2015
+
+  ## @param: horizon_url
+  ## @type: string
+  ## A horizon server endpoint.
+  ## @required_secret:
+  ##     PAYMENT_GATEWAY_STELLAR_SECRET_KEY - stellar account secret key
+  #
+  horizon_url: https://horizon-testnet.stellar.org
+
+
+callback_api:
+  ## @param: baseUrl
+  ## @type: string
+  ## @supported_prefixes: http:, https:
+  ## The anchor server callback API endpoint for customer, fee, quote, and unique url integration.
+  #
+  base_url: http://localhost:8081
+
+  ## Authentication config for Anchor server and Anchor Platform server to safely communicate,
+  ## particularly when housed in different clusters.
+  ## The receiving party should verify that an incoming request token is still valid.
+  #
+  auth:
+    ## @param: type
+    ## @supported_values: NONE, API_KEY, JWT_TOKEN
+    ##     NONE: no authentication is used
+    ##     API_KEY: The authentication is done using an API key added to the `X-Api-Key` header.
+    ##     JWT_TOKEN: The authentication is done using a JWT token added to the `Authorization`
+    ##         header. This token is generated from the secret key.
+    ##
+    ## If the type is not NONE, the secret must be set by the
+    ## environment variable: `secret.callback_api.auth_secret`
+    ##
+    ## The value of the secret depends on the type specified.
+    ##
+    ##    If type is JWT_TOKEN - the secret key of JWT encryption
+    ##    If type is API_KEY: - CallbackAPI requests (`Platform->Anchor`) will contain an
+    ##        authentication header whose token was built using this secret. The Anchor Backend will
+    ##        also store this same secret and use it to decode the incoming token to verify it
+    ##        came from the Platform.
+    #
+    type: NONE
+
+    ## @param: expiration_milliseconds
+    ## @type: int
+    ## Expiration time, in milliseconds, that will be used to build and validate the JWT tokens
+    #
+    expiration_milliseconds: 30000
+
+platform_api:
```

Comment:
Are we missing an `expiration_milliseconds` attribute like the `callback_api.auth` object has?

Reply:
Thanks for catching this. It was missing and now added.